networking_ccloud/ml2/agent/eos/switch.py

```diff
@@ -40,22 +40,28 @@ class EOSGNMIPaths:
     VLAN = "network-instances/network-instance[name=default]/vlans/vlan[vlan-id={vlan}]"
 
     VXMAPS = "interfaces/interface[name=Vxlan1]/arista-exp-eos-vxlan:arista-vxlan/config/vlan-to-vnis"
+    VXMAPS_ALL_4_32 = ("interfaces/interface[name=Vxlan1]/arista-exp-eos-vxlan:arista-vxlan/vlan-to-vnis/"
+                       "vlan-to-vni/config")
+    VXMAPS_CONFIG_4_32 = "interfaces/interface[name=Vxlan1]/arista-exp-eos-vxlan:arista-vxlan/vlan-to-vnis/"
     VXMAP_VLAN = ("interfaces/interface[name=Vxlan1]/arista-exp-eos-vxlan:arista-vxlan/config/vlan-to-vnis"
                   "/vlan-to-vni[vlan={vlan}]")
+    VXMAP_VLAN_4_32 = ("interfaces/interface[name=Vxlan1]/arista-exp-eos-vxlan:arista-vxlan/vlan-to-vnis"
+                       "/vlan-to-vni[vlan={vlan}]")
```

Comment:
Will we ever supporting more versions simultaneously?  This will obviously work, but may be doing encoding these paths in a dict keyed by the version might be an option if we ever go for that cases (probably less performant on the other hand).

Reply:
Looking at the code below that doesn't solve the problem many if-else blocks you would still have though

Reply:
Yeah, we most likely will need to support different versions, but most of the paths will not be keyed to a specific version. It will most likely be "before breaking the path" and "after breaking the path". In some cases we also need to revise the logic of how we construct the data, so it will most likely stay flag-based or we would need to write different methods that create certain parts of the config (maybe with decorators deciding for which version they are?).

Reply:
Yeah, that sounds reasonable. Probably not something strictly necessary for this PR